### PR TITLE
Javascript Function Applied to Menus

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Java script function applied to the top menu bar, that pops up the menu and hides it on clicking the cross button.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const hBtn = document.getElementById("menubar");
+const nav = document.querySelector("nav");
+function toggleMenu() {
+    hBtn.classList.toggle("open");
+    nav.classList.toggle("open");
+}
+
+hBtn.addEventListener("click", toggleMenu);
```
Comment (omarsalem7) [inline script.js:8]: - [ ] Consider when the clicks any option of the mobile menu, the mobile menu disappear.
- [ ] Kindly follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) to match the design like it and choose the correct font, colors, margins ,and padding.

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image|

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -14,6 +14,191 @@ body {
...
+  width: 100%;
+}
+
+#mobNav {
+  background-color: rgba(96, 112, 255, 1);
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(3px);
+  width: 0%;
+  height: 100%;
+  position: absolute;
```
Comment (leonardodiasb) [inline style.css:36]: - [OPTIONAL] Consider using position fixed for this mobile menu. If the user tries to scroll down while the menu is open, the menu can disappear from the screen.
- Your design:
image

Comment (AsimKhan2019) [reply style.css:36]: @leonardodiasb Thank you for the approval

**Pull request merged**